biome.jsonc

```diff
@@ -1,8 +1,6 @@
 {
-  "$schema": "https://biomejs.dev/schemas/1.7.1/schema.json",
-  "organizeImports": {
-    "enabled": false
-  },
+  "$schema": "https://biomejs.dev/schemas/2.2.4/schema.json",
+  "assist": { "actions": { "source": { "organizeImports": "off" } } },
```

Comment:
It generated by migration tool
ref: https://biomejs.dev/guides/upgrade-to-biome-v2/#the-import-organizer-sorts-the-imports-differently